MapboxNavigation/RouteManeuverViewController.swift

```diff
@@ -132,7 +132,9 @@ class RouteManeuverViewController: UIViewController {
             destinationLabel.unabridgedText = routeProgress.currentLeg.destination.name ?? routeStepFormatter.string(for: routeStepFormatter.string(for: routeProgress.currentLegProgress.upComingStep, legIndex: routeProgress.legIndex, numberOfLegs: routeProgress.route.legs.count, markUpWithSSML: false))
         } else if let upComingStep = routeProgress.currentLegProgress?.upComingStep {
             updateStreetNameForStep()
-            showLaneView(step: upComingStep)
+            if routeProgress.currentLegProgress.alertUserLevel == .medium || routeProgress.currentLegProgress.alertUserLevel == .high {
```

Comment:
I agree about the scrolling ahead.

However, let's assume we're going from one step to another (w/o swiping) which currently is at low alert level.
The page changes, `(willTransitionTo:)` gets called which in turn shows the lane view unconditionally. I think this is the case and also exactly what this PR is trying to avoid?